knexfile.js

```diff
@@ -12,12 +23,31 @@ module.exports = {
     port: DatabaseConfig.port,
     charset: 'utf8'
   },
-  useNullAsDefault: true,
-  migrations: {
-    tableName: 'knex_migrations',
-    directory: './db/migrations'
+  ...defaultConfig
```

Comment:
`...defaultConfig` should be the first item in the object so it doesn't override anything else we set.